jest.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     testEnvironment: './jest-test-environment.js',
 
     // Indicates whether each individual test should be reported during the run
-    verbose: true,
+    verbose: false,
```

Comment:
The verbose logs made it hard to find the correct log messages, especially from the input classifier test suite.